src/main/resources/default/taglib/t/heading.html.pasta

```diff
@@ -1,17 +1,17 @@
 <i:arg type="String" name="labelKey" default=""/>
 <i:arg type="String" name="label" default="@i18n(labelKey)"/>
-<i:arg type="String" name="class" default="" />
-<i:local name="markupLabel" value="@renderToString('body')" />
+<i:arg type="String" name="class" default=""/>
+<i:local name="markupLabel" value="@renderToString('body')"/>
 <i:if test="isFilled(label)">
```

Comment:
I personally prefer some blank lines here, but that is – of course – a matter of taste.

```suggestion
<i:arg type="String" name="class" default=""/>

<i:local name="markupLabel" value="@renderToString('body')"/>

<i:if test="isFilled(label)">
```